web-app/js/chosen/chosen.jquery.js

```diff
@@ -2,18 +2,19 @@
 Chosen, a Select Box Enhancer for jQuery and Prototype
 by Patrick Filler for Harvest, http://getharvest.com
 
-Version 1.1.0
+Version 1.8.7
```

Comment:
Is there option to remove all third party libs and install these during build?

Reply:
There is a way to do it with Grails 1 and 2 but it would be a huge refactoring so I wanted to wait until we moved to Grails 3.